DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/CourseSetup/CourseContentViewModelTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.TrackingSystem.CourseSetup
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models;
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseContent;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    internal class CourseContentViewModelTests
+    {
+        private static readonly Tutorial DisabledTutorial = TutorialTestHelper.GetDefaultTutorial();

```

Comment:
Nice tests! Just a minor thing - it might be more intuitive if "GetDefaultTutorial" returned an enabled tutorial by default